include/sound/sof/topology.h

```diff
@@ -63,8 +63,7 @@ struct sof_ipc_comp {
 	uint32_t pipeline_id;
 	uint32_t core;
 
-	/* reserved for future use */
-	uint32_t reserved[1];
+	uint32_t subtype;	/**< flavour for generic component type */
 } __packed;
```

Comment:
The FW discussion ( https://github.com/thesofproject/sof/pull/2956 ) seems to be moving to use UUIDs directly, so let's target that.

Reply:
Okay.